### DISCUSSION TITLE
How to install openNCA

### DISCUSSION THREAD

**Discussion by yanxiaoyu2001** [tensfeldt/openNCA#282] [Q&A]
Dear Tom,

Could you help to provide some guidance on how can this openNCA R package be installed?

Thanks,
Xiaoyu

**Comment by tensfeldt** [comment]
@yanxiaoyu2001 I'm so pleased you have interest in this project.

I'm working this week on cleaning up the git history associated with the 3 releases we've had with openNCA computation engine. As soon as that is complete, I'll provide an vignette that will provide guidance on installation and use.

Can you provide some context on your specific use case so I can be sure to cover that appropriately?

Tom Tensfeldt
Sr Dir, Clinical Pharmacology and Pharmacometrics, Pfizer

**Reply by yanxiaoyu2001** [reply]
Dear Tom,

Many thanks for the prompt reply! I loofward to the guidance and installation. I am interested in using it for analyzing some PK data after oral administration. 

I watched your nice talk on POSSC. It seems that Pfizer's internal tool has lots of other cool features like audit trail, SHINY app, report generation, etc. Are those also written in R?

Best,
Xiaoyu

**Reply by tensfeldt** [reply]
Xiaoyu, I'm still working through getting the repository history cleaned up and a release prepared with instructions for installation and use. Thanks for your patience. If you have any questions while I get that sorted, please let me know. I'd encourage you to take a look at the [computation engine specifications document](https://github.com/tensfeldt/openNCA/blob/master/openNCA/man/openNCA_Computation_Engine_Specification.docx) (currently stored in master branch but may move around as I clean this up).
Happy to try to answer any questions in the interim. Note eventually, we're also looking to open source the opennca application which uses the R package computation engine. More info to come...
Tom Tensfeldt
Sr Dir, Clinical Pharmacology and Pharmacometrics, Pfizer

**Reply by yanxiaoyu2001** [reply]
Tom, Many thanks for the note. I look forward to it. Xiaoyu
